src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -142,6 +145,8 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
     this.audioVideoController = undefined;
     this.contentShareController = undefined;
     this._eventReporter = undefined;
+
+    CSPViolationMonitor.disableCSPMonitor();
```

Comment:
This isn't right. We want the CSP monitor to stick around forever, unless the builder deliberately turns it off — what you have here will cause the CSP monitor to be removed after the first meeting is destroyed. 

What you _do_ need a way to do is to remove the logger a few lines above.

Reply:
Make sense. Will remove.